frontend/packages/topology/src/components/export-app/ExportApplication.tsx

```diff
@@ -0,0 +1,126 @@
+import { ToolbarItem, Button, AlertVariant } from '@patternfly/react-core';
+import * as _ from 'lodash';
+import { useTranslation, Trans } from 'react-i18next';
+import { useAccessReview } from '@console/internal/components/utils';
+import { dateTimeFormatter } from '@console/internal/components/utils/datetime';
+import { k8sCreate, k8sGet, k8sKill, K8sResourceKind } from '@console/internal/module/k8s';
+import {
+  useFlag,
+  useIsMobile,
+  USERSETTINGS_PREFIX,
+  useToast,
+  useUserSettings,
+} from '@console/shared/src';
+import { ALLOW_EXPORT_APP, EXPORT_CR_NAME } from '../../const';
+import { ExportModel } from '../../models';
+import { getExportAppData } from '../../utils/export-app-utils';
+import exportApplicationModal from './export-app-modal';
+import { ExportAppUserSettings } from './types';
+
+type ExportApplicationProps = {
+  namespace: string;
+  isDisabled: boolean;
+};
+
+const ExportApplication: React.FC<ExportApplicationProps> = ({ namespace, isDisabled }) => {
+  const { t } = useTranslation();
+  const [isCreating, setIsCreating] = React.useState<boolean>(false);
+  const isMobile = useIsMobile();
+  const isExportAppAllowed = useFlag(ALLOW_EXPORT_APP);
+  const canExportApp = useAccessReview({
+    group: ExportModel.apiGroup,
+    resource: ExportModel.plural,
+    verb: 'create',
+    namespace,
+  });
+  const toast = useToast();
+  const [exportAppToast, setExportAppToast] = useUserSettings<ExportAppUserSettings>(
+    `${USERSETTINGS_PREFIX}.exportApp`,
+    {},
+    true,
+  );
+
+  const createExportCR = async () => {
+    try {
+      const exportResp = await k8sCreate<K8sResourceKind>(ExportModel, getExportAppData(namespace));
+      const key = `${namespace}-${exportResp.metadata.name}`;
+      const exportAppToastConfig = {
+        ...exportAppToast,
+        [key]: {
+          uid: exportResp.metadata.uid,
+          name: exportResp.metadata.name,
+          kind: exportResp.kind,
+          namespace,
+        },
+      };
+      toast.addToast({
+        variant: AlertVariant.info,
+        title: t('topology~Export Application'),
+        content: (
+          <Trans t={t} ns="topology">
+            Export of resources in <strong>{{ namespace }}</strong> has started.
+          </Trans>
+        ),
+        dismissible: true,
+        timeout: true,
+      });
+      setExportAppToast(exportAppToastConfig);
+      setIsCreating(false);
+    } catch (error) {
+      setIsCreating(false);
+      toast.addToast({
+        variant: AlertVariant.danger,
+        title: t('topology~Export Application'),
+        content: (
+          <Trans t={t} ns="topology">
+            Export of resources in <strong>{{ namespace }}</strong> has failed with error:{' '}
+            {error.message}
+          </Trans>
+        ),
+        dismissible: true,
+        timeout: true,
+      });
+    }
+  };
+
+  const exportAppClickHandle = async () => {
+    try {
+      setIsCreating(true);
```

Comment:
Start an export and then click the export button again to see the modal.
In another window delete the export resource. You will be unable to start the export in the first window again because this flag is set.